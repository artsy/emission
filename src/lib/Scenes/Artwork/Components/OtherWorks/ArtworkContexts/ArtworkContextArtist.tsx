import { ArtworkContextArtist_artwork } from "__generated__/ArtworkContextArtist_artwork.graphql"
import React from "react"
import { createFragmentContainer, graphql } from "react-relay"

import { ContextSectionsContainer } from "../ContextSectionsContainer"
import { ArtistArtworkGrid, PartnerArtworkGrid, RelatedArtworkGrid } from "./ArtworkGrids"

export const ArtworkContextArtistFragmentContainer = createFragmentContainer<{
  artwork: ArtworkContextArtist_artwork
}>(
  props => {
    return (
      <ContextSectionsContainer>
        <ArtistArtworkGrid artwork={props.artwork} />
        <PartnerArtworkGrid artwork={props.artwork} />
        <RelatedArtworkGrid artwork={props.artwork} />
      </ContextSectionsContainer>
    )
  },
  {
    artwork: graphql`
      fragment ArtworkContextArtist_artwork on Artwork {
        ...ArtistArtworkGrid_artwork
        ...PartnerArtworkGrid_artwork
        ...RelatedArtworkGrid_artwork
      }
    `,
  }
)
