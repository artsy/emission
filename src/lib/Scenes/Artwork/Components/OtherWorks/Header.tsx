import { Flex, Serif } from "@artsy/palette"
import React from "react"

interface HeaderProps {
  children?: JSX.Element
  title: string
}

export const Header: React.SFC<HeaderProps> = props => {
  const { children, title } = props

  return (
    <Flex flexDirection="column" alignItems="center">
      <Serif size="5t" mb={2} textAlign="center">
        {title}
      </Serif>
      {children}
    </Flex>
  )
}
