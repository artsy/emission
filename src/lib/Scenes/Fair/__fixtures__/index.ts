export const fairFixture = {
  slug: "sofa-chicago-2018",
  internalID: "5aa1a9ee8b3b817c61dff02d",
  followed_content: null,
  name: "SOFA CHICAGO 2018",
  exhibition_period: "Nov 1 – 4, 2018",
  image: {
    image_url: "https://d32dm0rphc51dk.cloudfront.net/o0Zrtm-CgwbV1syD_33DuA/:version.jpg",
    aspect_ratio: 1,
    url: "https://d32dm0rphc51dk.cloudfront.net/o0Zrtm-CgwbV1syD_33DuA/square.jpg",
  },
  sponsoredContent: {
    activationText: "This fair will be incroyable. Come check out why.",
    pressReleaseUrl: "https://www.somefair.com",
  },
  profile: {
    slug: "art-basel-in-miami-beach-2018",
    internalID: "dfdlfjfua7d8a0_",
    is_followed: false,
    name: "art-basel",
    icon: {
      internalID: "58ff95298b3b8104783c119c",
      href: null,
      height: 279,
      width: 279,
      url: "https://d32dm0rphc51dk.cloudfront.net/B1x-CqoWO1KitQnRj7PrrA/untouched-png.png",
    },
  },
  organizer: {
    profile: {
      icon: {
        internalID: "58ff95298b3b8104783c119c",
        href: "",
        height: 279,
        width: 279,
        url: "https://d32dm0rphc51dk.cloudfront.net/B1x-CqoWO1KitQnRj7PrrA/untouched-png.png",
      },
      name: "SOFA CHICAGO",
      id: "UHJvZmlsZTpzb2ZhLWNoaWNhZ28=",
    },
    website: "",
  },
  start_at: "2018-11-01T17:00:00+00:00",
  end_at: "2018-11-04T18:00:00+00:00",
  formattedOpeningHours: "Opens Nov 1 at 5:00pm",
  location: {
    id: "TG9jYXRpb246NWJiNjJmMzYyYjVlN2QwMDI5ODliZDhh",
    internalID: "5bb62f362b5e7d002989bd8a",
    address_2: "",
    postal_code: "60611",
    summary: "Festival Hall, Navy Pier \r\n600 East Grand Avenue Chicago, IL 60611",
    address: "",
    city: "Chicago",
    coordinates: {
      lat: 25.7949009,
      lng: -80.134698,
    },
    cached: null,
    day_schedules: null,
    day_schedule_text: null,
    openingHours: {
      schedules: [
        {
          days: "Monday–Sunday",
          hours: "Closed",
        },
      ],
    },
    display: "25.794890, -80.134550",
  },
  counts: {
    artists: 809,
    artworks: 2028,
    partners: 127,
    partner_shows: 127,
  },
  exhibitors_grouped_by_name: [
    {
      letter: "3",
      exhibitors: [
        {
          name: "303 Gallery",
          slug: "303 Gallery",
          profile_id: "303 Gallery",
        },
      ],
    },
    {
      letter: "A",
      exhibitors: [
        { name: "Acquavella Galleries", slug: "Acquavella Galleries", profile_id: "Acquavella Galleries" },
        { name: "Alan Cristea Gallery", slug: "Alan Cristea Gallery", profile_id: "Alan Cristea Gallery" },
        { name: "Alfonso Artiaco", slug: "Alfonso Artiaco", profile_id: "Alfonso Artiaco" },
        { name: "Altman Siegel", slug: "Altman Siegel", profile_id: "Altman Siegel" },
        { name: "Anat Ebgi", slug: "Anat Ebgi", profile_id: "Anat Ebgi" },
        { name: "Annely Juda Fine Art", slug: "Annely Juda Fine Art", profile_id: "Annely Juda Fine Art" },
        { name: "Anton Kern Gallery", slug: "Anton Kern Gallery", profile_id: "Anton Kern Gallery" },
      ],
    },
    {
      letter: "B",
      exhibitors: ["Bergamin & Gomide", "Berggruen Gallery", "Blum & Poe", "Boers-Li Gallery", "Bortolami"],
    },
    {
      letter: "C",
      exhibitors: [
        "CARDI GALLERY",
        "Casa Triângulo",
        "Casey Kaplan",
        "Cheim & Read",
        "Cristina Guerra Contemporary Art",
        "Crown Point Press",
      ],
    },
    {
      letter: "D",
      exhibitors: ["Dan Galeria", "DC Moore Gallery"],
    },
    {
      letter: "E",
      exhibitors: ["Edward Tyler Nahem Fine Art LLC", "Edwynn Houk Gallery", "Eric Firestone Gallery"],
    },
    {
      letter: "F",
      exhibitors: ["Fergus McCaffrey", "Fortes D'Aloia & Gabriel", "Francis M. Naumann Fine Art"],
    },
    {
      letter: "G",
      exhibitors: [
        "Gagosian",
        "Galerie Chantal Crousel",
        "Galerie Christophe Gaillard",
        "galerie frank elbaz",
        "GALERIE GEORGES-PHILIPPE ET NATHALIE VALLOIS",
        "Galerie Greta Meert",
        "Galerie Jérôme Poggi",
        "GALLERIA CONTINUA",
        "Gemini G.E.L.",
        "Gladstone Gallery",
        "Goodman Gallery",
        "GRIMM",
      ],
    },
    {
      letter: "H",
      exhibitors: ["Haines Gallery", "Hales Gallery", "Hammer Galleries", "Hauser & Wirth", "Howard Greenberg Gallery"],
    },
    {
      letter: "I",
      exhibitors: ["Ingleby Gallery", "Inman Gallery", "Instituto de Visión"],
    },
    {
      letter: "J",
      exhibitors: ["James Cohan"],
    },
    {
      letter: "K",
      exhibitors: ["Kasmin", "Kavi Gupta", "Kayne Griffin Corcoran", "KEWENIG"],
    },
    {
      letter: "M",
      exhibitors: ["Marian Goodman Gallery", "Marianne Boesky Gallery"],
    },
    {
      letter: "P",
      exhibitors: ["Peter Blum Gallery", "Pilar Corrias Gallery"],
    },
    {
      letter: "R",
      exhibitors: ["Rhona Hoffman Gallery", "Richard Gray Gallery"],
    },
    {
      letter: "S",
      exhibitors: ["Sean Kelly Gallery"],
    },
    {
      letter: "X",
      exhibitors: ["Xavier Hufkens"],
    },
  ],
  filteredArtworks: {
    aggregations: [
      {
        slice: "PRICE_RANGE",
        counts: [
          {
            id: "*-*",
            name: "for Sale",
          },
          {
            id: "1000-5000",
            name: "between $1,000 & $5,000",
          },
          {
            id: "5000-10000",
            name: "between $5,000 & $10,000",
          },
          {
            id: "10000-50000",
            name: "between $10,000 & $50,000",
          },
          {
            id: "*-1000",
            name: "Under $1,000",
          },
          {
            id: "50000-*",
            name: "over $50,000",
          },
        ],
      },
      {
        slice: "MEDIUM",
        counts: [
          {
            id: "sculpture",
            name: "Sculpture",
          },
          {
            id: "painting",
            name: "Painting",
          },
          {
            id: "design",
            name: "Design",
          },
          {
            id: "photography",
            name: "Photography",
          },
          {
            id: "jewelry",
            name: "Jewelry",
          },
          {
            id: "work-on-paper",
            name: "Work on Paper",
          },
          {
            id: "installation",
            name: "Installation",
          },
          {
            id: "prints",
            name: "Prints",
          },
          {
            id: "drawing",
            name: "Drawing",
          },
          {
            id: "film-slash-video",
            name: "Film/Video",
          },
        ],
      },
    ],
    artworks_connection: {
      edges: [
        {
          node: {
            slug: "watanabe-chiaki-links",
            title: "Links",
            date: "2018",
            sale_message: "Sold",
            is_in_auction: false,
            is_biddable: false,
            is_acquireable: false,
            sale: null,
            sale_artwork: null,
            image: {
              url: "https://d32dm0rphc51dk.cloudfront.net/uF3UWRba84Q4wSrRB-BxKA/large.jpg",
              aspect_ratio: 1.61,
            },
            artists: [
              {
                name: "Watanabe Chiaki",
              },
            ],
            partner: {
              name: "TAI Modern",
            },
            href: "/artwork/watanabe-chiaki-links",
          },
        },
        {
          node: {
            slug: "dylan-martinez-water-bag-number-34",
            title: "Water Bag #34",
            date: "2018",
            sale_message: "$1,800",
            is_in_auction: false,
            is_biddable: false,
            is_acquireable: false,
            sale: null,
            sale_artwork: null,
            image: {
              url: "https://d32dm0rphc51dk.cloudfront.net/EB-WA2El6g-tII076F5vFw/large.jpg",
              aspect_ratio: 0.67,
            },
            artists: [
              {
                name: "Dylan Martinez",
              },
            ],
            partner: {
              name: "Lyons Wier Gallery",
            },
            href: "/artwork/dylan-martinez-water-bag-number-34",
          },
        },
        {
          node: {
            slug: "dylan-martinez-water-bag-number-23",
            title: "Water Bag #23",
            date: "2018",
            sale_message: "$1,800",
            is_in_auction: false,
            is_biddable: false,
            is_acquireable: false,
            sale: null,
            sale_artwork: null,
            image: {
              url: "https://d32dm0rphc51dk.cloudfront.net/sAKQugZCVUZ9g5AoVyCJdA/large.jpg",
              aspect_ratio: 0.67,
            },
            artists: [
              {
                name: "Dylan Martinez",
              },
            ],
            partner: {
              name: "Lyons Wier Gallery",
            },
            href: "/artwork/dylan-martinez-water-bag-number-23",
          },
        },
        {
          node: {
            slug: "dylan-martinez-water-bag-number-32",
            title: "Water Bag #32",
            date: "2018",
            sale_message: "$1,800",
            is_in_auction: false,
            is_biddable: false,
            is_acquireable: false,
            sale: null,
            sale_artwork: null,
            image: {
              url: "https://d32dm0rphc51dk.cloudfront.net/jVS6Bolng37bN8GB0y8qfQ/large.jpg",
              aspect_ratio: 0.67,
            },
            artists: [
              {
                name: "Dylan Martinez",
              },
            ],
            partner: {
              name: "Lyons Wier Gallery",
            },
            href: "/artwork/dylan-martinez-water-bag-number-32",
          },
        },
        {
          node: {
            slug: "dylan-martinez-mach-2",
            title: "Mach 2",
            date: "2018",
            sale_message: "$12,000",
            is_in_auction: false,
            is_biddable: false,
            is_acquireable: false,
            sale: null,
            sale_artwork: null,
            image: {
              url: "https://d32dm0rphc51dk.cloudfront.net/tnjfaBzOFmFvfj227QjLJg/large.jpg",
              aspect_ratio: 1.5,
            },
            artists: [
              {
                name: "Dylan Martinez",
              },
            ],
            partner: {
              name: "Lyons Wier Gallery",
            },
            href: "/artwork/dylan-martinez-mach-2",
          },
        },
        {
          node: {
            slug: "dylan-martinez-water-bag-trio",
            title: "Water Bag Trio",
            date: "2018",
            sale_message: "$5,400",
            is_in_auction: false,
            is_biddable: false,
            is_acquireable: false,
            sale: null,
            sale_artwork: null,
            image: {
              url: "https://d32dm0rphc51dk.cloudfront.net/BwEuuw_gYmFs69ENusdang/large.jpg",
              aspect_ratio: 0.67,
            },
            artists: [
              {
                name: "Dylan Martinez",
              },
            ],
            partner: {
              name: "Lyons Wier Gallery",
            },
            href: "/artwork/dylan-martinez-water-bag-trio",
          },
        },
      ],
    },
  },
  artists: {
    edges: [
      {
        node: {
          href: "/tamas",
          slug: "tamas-abel",
          internalID: "1232222245",
          id: "QXJ0aXN0OnRhbWFzLWFiZWw=",
          name: "Tamás Ábel",
          is_followed: false,
          nationality: "Hungarian",
          birthday: "",
          deathday: "",
          image: {
            url: "https://d32dm0rphc51dk.cloudfront.net/mSFWvVbcQezxjjS0ordckg/large.jpg",
          },
        },
      },
      {
        node: {
          href: "/charlie",
          slug: "charlie-aldrich",
          internalID: "12347775",
          id: "QXJ0aXN0OmNoYXJsaWUtYWxkcmljaA==",
          name: "Charlie Aldrich",
          is_followed: false,
          nationality: "United States",
          birthday: "",
          deathday: "",
          image: {
            url: "https://d32dm0rphc51dk.cloudfront.net/3v-35X7zWI3p6Lk8p6pYhg/large.jpg",
          },
        },
      },
      {
        node: {
          href: "/rik",
          slug: "rik-allen",
          internalID: "123477750000000000",
          id: "QXJ0aXN0OnJpay1hbGxlbg==",
          name: "Rik Allen",
          is_followed: false,
          nationality: "",
          birthday: "",
          deathday: "",
          image: {
            url: "https://d32dm0rphc51dk.cloudfront.net/8Irjs-pCGNU9VvWg9r8wFg/tall.jpg",
          },
        },
      },
      {
        node: {
          href: "/jean",
          slug: "jean-luc-almond",
          internalID: "1232222245111",
          id: "QXJ0aXN0OmplYW4tbHVjLWFsbW9uZA==",
          name: "Jean-Luc Almond",
          is_followed: false,
          nationality: "English",
          birthday: "",
          deathday: "",
          image: {
            url: "https://d32dm0rphc51dk.cloudfront.net/qDxLZGl8BHeP8zCkoeW8tw/tall.jpg",
          },
        },
      },
      {
        node: {
          href: "/amador",
          slug: "amador",
          internalID: "123222224588888",
          id: "QXJ0aXN0OmFtYWRvcg==",
          name: "Amador",
          is_followed: false,
          nationality: "Spanish",
          birthday: "1957",
          deathday: "",
          image: {
            url: "https://d32dm0rphc51dk.cloudfront.net/UEPyf29fi9y4wwaHTJ9nqg/large.jpg",
          },
        },
      },
    ],
  },
  id: "RmFpcjpzb2ZhLWNoaWNhZ28tMjAxOA==",
  shows_connection: {
    edges: [
      {
        node: {
          slug: "spruth-magers",
          partner: {
            __typename: "Partner",
            profile: {
              slug: "spruth-magers",
              href: "/spruth-magers",
              name: "Sprüth Magers",
              internalID: "123456",
            },
          },
        },
      },
      {
        node: {
          slug: "cardi-gallery",
          partner: {
            __typename: "Partner",
            profile: {
              slug: "cardi-gallery",
              href: "/cardi-gallery",
              name: "CARDI GALLERY",
              internalID: "1234562",
            },
          },
        },
      },
    ],
  },
  shows: {
    pageInfo: {
      endCursor: "0.7605261301690746:5ba9f83787830600062e7f86",
      startCursor: "YXJyYXljb25uZWN0aW9uOjA=",
      hasNextPage: true,
      hasPreviousPage: false,
    },
    edges: [
      {
        cursor: "YXJyYXljb25uZWN0aW9uOjA=",
        node: {
          slug: "steidel-contemporary-steidel-contemporary-at-sofa-chicago-2018",
          internalID: "111",
          counts: {
            artworks: 4,
          },
          name: "Steidel Contemporary at SOFA CHICAGO 2018",
          is_fair_booth: true,
          partner: {
            id: "UGFydG5lcjpzdGVpZGVsLWNvbnRlbXBvcmFyeQ==",
            internalID: "123123123123123123123",
            slug: "steidel-contemporary",
            name: "Steidel Contemporary",
            href: "http://www.stei.com",
            profile: {
              is_followed: false,
              internalID: "1231231231",
            },
          },
          fair: {
            name: "SOFA CHICAGO 2018",
            id: "RmFpcjpzb2ZhLWNoaWNhZ28tMjAxOA==",
          },
          cover_image: {
            url: "https://d32dm0rphc51dk.cloudfront.net/nltwPbizCU7VKKrjmEiqUA/medium.jpg",
          },
          location: {
            display: "Booth B32",
            id: "TG9jYXRpb246NWI5ODIwNjU0OTUzZjEwMDA1ZjY5ZTBh",
          },
          profile: {
            slug: "art-basel-in-miami-beach-2018",
          },
          artworks_connection: {
            edges: [
              {
                node: {
                  slug: "christine-flynn-costa-rica-palms-1",
                  title: "Costa Rica Palms",
                  date: "2018",
                  sale_message: "£2,500 - 5,000",
                  is_in_auction: false,
                  is_biddable: false,
                  is_acquireable: false,
                  is_offerable: false,
                  sale: null,
                  sale_artwork: null,
                  href: "/artwork/christine-flynn-costa-rica-palms-1",
                  artists: [
                    {
                      slug: "christine-flynn-1",
                      name: "Christine Flynn",
                    },
                  ],
                  partner: {
                    slug: "kahn-gallery",
                    internalID: "58d3ac7dc9dc241448f6ec7c",
                    name: "Kahn Gallery",
                  },
                  image: {
                    url: "https://d32dm0rphc51dk.cloudfront.net/sxzp6tLM3zFJ3dz5kfIqVA/larger.jpg",
                    aspect_ratio: 1.64,
                  },
                },
              },
              {
                node: {
                  slug: "reinhard-gorner-palacio-nacional-de-mafra-portugal",
                  title: "Palacio Nacional de Mafra, Portugal",
                  date: "2016",
                  sale_message: "$5,500",
                  is_in_auction: false,
                  is_biddable: false,
                  is_acquireable: false,
                  is_offerable: false,
                  sale: null,
                  sale_artwork: null,
                  href: "/artwork/reinhard-gorner-palacio-nacional-de-mafra-portugal",
                  artists: [
                    {
                      slug: "reinhard-gorner",
                      name: "Reinhard Gorner",
                    },
                  ],
                  partner: {
                    slug: "kahn-gallery",
                    internalID: "58d3ac7dc9dc241448f6ec7c",
                    name: "Kahn Gallery",
                  },
                  image: {
                    url: "https://d32dm0rphc51dk.cloudfront.net/nkFRIEsLDAZPUQkcN7Nb3Q/small.jpg",
                    aspect_ratio: 0.88,
                  },
                },
              },
              {
                node: {
                  slug: "elisabeth-lecourt-ma-poupee-de-cire-en-fanfaron-nepal",
                  title: "Ma Poupée de Cire en Fanfaron Népal",
                  date: "2018",
                  sale_message: "Sold",
                  is_in_auction: false,
                  is_biddable: false,
                  is_acquireable: false,
                  is_offerable: false,
                  sale: null,
                  sale_artwork: null,
                  href: "/artwork/elisabeth-lecourt-ma-poupee-de-cire-en-fanfaron-nepal",
                  artists: [
                    {
                      slug: "elisabeth-lecourt",
                      name: "Elisabeth LECOURT",
                    },
                  ],
                  partner: {
                    slug: "kahn-gallery",
                    internalID: "58d3ac7dc9dc241448f6ec7c",
                    name: "Kahn Gallery",
                  },
                  image: {
                    url: "https://d32dm0rphc51dk.cloudfront.net/GJPv6LnC3XpZasK1C3rzmQ/larger.jpg",
                    aspect_ratio: 0.74,
                  },
                },
              },
              {
                node: {
                  slug: "damien-gard-au-travers-des-siecles-1",
                  title: "Au Travers des Siècles",
                  date: "2017",
                  sale_message: "£2,500 - 5,000",
                  is_in_auction: false,
                  is_biddable: false,
                  is_acquireable: false,
                  is_offerable: false,
                  sale: null,
                  sale_artwork: null,
                  href: "/artwork/damien-gard-au-travers-des-siecles-1",
                  artists: [
                    {
                      slug: "damien-gard",
                      name: "Damien Gard",
                    },
                  ],
                  partner: {
                    slug: "kahn-gallery",
                    internalID: "58d3ac7dc9dc241448f6ec7c",
                    name: "Kahn Gallery",
                  },
                  image: {
                    url: "https://d32dm0rphc51dk.cloudfront.net/SdvT-flDhUHGblmIDdsqtg/medium_rectangle.jpg",
                    aspect_ratio: 0.71,
                  },
                },
              },
            ],
          },
          id: "U2hvdzpzdGVpZGVsLWNvbnRlbXBvcmFyeS1zdGVpZGVsLWNvbnRlbXBvcmFyeS1hdC1zb2ZhLWNoaWNhZ28tMjAxOA==",
          __typename: "Show",
        },
      },
    ],
  },
  isActive: true,
  hours:
    "Thursday, November 1, 2018: 5 - 9 pm\r\nFriday, November 2, 2018: 11 am - 7 pm (10 - 11 am VIP hour)\r\nSaturday, November 3, 2018: 11 am - 7 pm (10 - 11 am VIP hour)\r\nSunday, November 4, 2018: 12 - 6 pm ",
  links:
    "[Palm Beach Modern + Contempoary](http://www.artpbfair.com/)\r\n[Facebook](https://www.facebook.com/PalmBeachModernandContemporary) | [Instagram](https://www.instagram.com/artmiamifairs/) | [Twitter](https://twitter.com/PB_Modern)",
  about:
    "Cillum adipisicing dolore qui ullamco deserunt ut in. In ad do laboris fugiat nostrud. Officia et duis sunt velit cupidatat irure.\r\n\r\nAdipisicing aliqua sunt magna nisi qui mollit non amet commodo ad ex eu. Nostrud nostrud excepteur sint esse occaecat qui et magna reprehenderit. Non do eiusmod ullamco consequat.\r\n\r\nCommodo exercitation aliqua nostrud minim sint laboris et. Fugiat elit labore magna Lorem aliqua ex nulla ea in sunt consequat. Fugiat tempor minim ex ex ullamco quis ullamco ea. Proident consectetur pariatur aliqua minim voluptate ad sunt sint aliqua mollit ullamco enim proident. Aliquip adipisicing elit ad mollit ad magna est officia nisi mollit eu. Cupidatat aute ex deserunt ad voluptate labore ipsum voluptate. Elit proident nisi labore deserunt cupidatat exercitation dolor nulla sint mollit in.",
  ticketsLink: "https://ticket.artbasel.com/MCH/miamibeach2018/Shop",
}
