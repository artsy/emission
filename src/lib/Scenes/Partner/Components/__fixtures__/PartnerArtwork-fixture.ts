export const PartnerArtworkFixture = {
  " $fragmentRefs": null as any,
  " $refType": null as any,
  internalID: "4d8b92c44eb68a1b2c0004cb",
  artworks: {
    edges: [
      {
        node: {
          id: "QXJ0d29yazo0ZDhiOTJjNzRlYjY4YTFiMmMwMDA1NDA=",
          image: {
            aspect_ratio: 0.71,
            url: "https://d32dm0rphc51dk.cloudfront.net/Wpq1gvhpupGvUcUwtdl0KQ/large.jpg",
          },
          title: "Untitled",
          date: "2005",
          sale_message: null,
          is_biddable: false,
          is_acquireable: false,
          is_offerable: false,
          slug: "cecily-brown-untitled-18",
          sale: null,
          sale_artwork: null,
          artists: [
            {
              name: "Cecily Brown",
            },
          ],
          partner: {
            name: "Gagosian",
          },
          href: "/artwork/cecily-brown-untitled-18",
        },
      },
      {
        node: {
          id: "QXJ0d29yazo0ZDhiOTJmYzRlYjY4YTFiMmMwMDBiNDI=",
          image: {
            aspect_ratio: 0.78,
            url: "https://d32dm0rphc51dk.cloudfront.net/0uqXV6WS2yR264w7vbxeZw/large.jpg",
          },
          title: "Purple Bra",
          date: "2006",
          sale_message: null,
          is_biddable: false,
          is_acquireable: false,
          is_offerable: false,
          slug: "john-currin-purple-bra",
          sale: null,
          sale_artwork: null,
          artists: [
            {
              name: "John Currin",
            },
          ],
          partner: {
            name: "Gagosian",
          },
          href: "/artwork/john-currin-purple-bra",
        },
      },
      {
        node: {
          id: "QXJ0d29yazo0ZDhiOTJmZTRlYjY4YTFiMmMwMDBiN2I=",
          image: {
            aspect_ratio: 1.56,
            url: "https://d32dm0rphc51dk.cloudfront.net/w9vKR48fw3DiE0ha5B5bOw/large.jpg",
          },
          title: "Untitled",
          date: "2007",
          sale_message: null,
          is_biddable: false,
          is_acquireable: false,
          is_offerable: false,
          slug: "gregory-crewdson-untitled-69",
          sale: null,
          sale_artwork: null,
          artists: [
            {
              name: "Gregory Crewdson",
            },
          ],
          partner: {
            name: "Gagosian",
          },
          href: "/artwork/gregory-crewdson-untitled-69",
        },
      },
      {
        node: {
          id: "QXJ0d29yazo0ZDhiOTJmNTRlYjY4YTFiMmMwMDBhNWU=",
          image: {
            aspect_ratio: 1.13,
            url: "https://d32dm0rphc51dk.cloudfront.net/U2_icKldX8zDuKmSGDoUlg/large.jpg",
          },
          title: "Snowman",
          date: "2004",
          sale_message: null,
          is_biddable: false,
          is_acquireable: false,
          is_offerable: false,
          slug: "mark-tansey-snowman",
          sale: null,
          sale_artwork: null,
          artists: [
            {
              name: "Mark Tansey",
            },
          ],
          partner: {
            name: "Gagosian",
          },
          href: "/artwork/mark-tansey-snowman",
        },
      },
      {
        node: {
          id: "QXJ0d29yazo0ZDhiOTMzYjRlYjY4YTFiMmMwMDExMWM=",
          image: {
            aspect_ratio: 0.78,
            url: "https://d32dm0rphc51dk.cloudfront.net/eKsH3Qe7Iu7dXQgOwOyn5g/large.jpg",
          },
          title: "Peg Leg",
          date: "2006",
          sale_message: null,
          is_biddable: false,
          is_acquireable: false,
          is_offerable: false,
          slug: "jeff-koons-peg-leg",
          sale: null,
          sale_artwork: null,
          artists: [
            {
              name: "Jeff Koons",
            },
          ],
          partner: {
            name: "Gagosian",
          },
          href: "/artwork/jeff-koons-peg-leg",
        },
      },
      {
        node: {
          id: "QXJ0d29yazo0ZDhiOTM1NTRlYjY4YTFiMmMwMDEzYWQ=",
          image: {
            aspect_ratio: 0.73,
            url: "https://d32dm0rphc51dk.cloudfront.net/VfN5PAsxyuaTfLz47h_Gjg/large.jpg",
          },
          title: "For They Know Not What They Do (Self-Portrait)",
          date: "2006",
          sale_message: null,
          is_biddable: false,
          is_acquireable: false,
          is_offerable: false,
          slug: "dan-colen-for-they-know-not-what-they-do-self-portrait",
          sale: null,
          sale_artwork: null,
          artists: [
            {
              name: "Dan Colen",
            },
          ],
          partner: {
            name: "Gagosian",
          },
          href: "/artwork/dan-colen-for-they-know-not-what-they-do-self-portrait",
        },
      },
      {
        node: {
          id: "QXJ0d29yazo0ZDhiOTM1MjRlYjY4YTFiMmMwMDEzNmU=",
          image: {
            aspect_ratio: 1.34,
            url: "https://d32dm0rphc51dk.cloudfront.net/vKzEFsW8gckSlWwkRWLSIQ/large.jpg",
          },
          title: "Untitled",
          date: "2007",
          sale_message: null,
          is_biddable: false,
          is_acquireable: false,
          is_offerable: false,
          slug: "dike-blair-untitled-10",
          sale: null,
          sale_artwork: null,
          artists: [
            {
              name: "Dike Blair",
            },
          ],
          partner: {
            name: "Gagosian",
          },
          href: "/artwork/dike-blair-untitled-10",
        },
      },
      {
        node: {
          id: "QXJ0d29yazo0ZDhiOTMzMDRlYjY4YTFiMmMwMDEwMmE=",
          image: {
            aspect_ratio: 0.6,
            url: "https://d32dm0rphc51dk.cloudfront.net/_CDPOHfm9KXnLyvQVCYaYQ/large.jpg",
          },
          title: "One Green Cow",
          date: "1979",
          sale_message: null,
          is_biddable: false,
          is_acquireable: false,
          is_offerable: false,
          slug: "andy-warhol-one-green-cow",
          sale: null,
          sale_artwork: null,
          artists: [
            {
              name: "Andy Warhol",
            },
          ],
          partner: {
            name: "Gagosian",
          },
          href: "/artwork/andy-warhol-one-green-cow",
        },
      },
      {
        node: {
          id: "QXJ0d29yazo0ZDhiOTM2NzRlYjY4YTFiMmMwMDE1MjA=",
          image: {
            aspect_ratio: 1,
            url: "https://d32dm0rphc51dk.cloudfront.net/BOXfMix8m119pln0iWe78w/large.jpg",
          },
          title: "Duet",
          date: "2004",
          sale_message: null,
          is_biddable: false,
          is_acquireable: false,
          is_offerable: false,
          slug: "mark-tansey-duet",
          sale: null,
          sale_artwork: null,
          artists: [
            {
              name: "Mark Tansey",
            },
          ],
          partner: {
            name: "Gagosian",
          },
          href: "/artwork/mark-tansey-duet",
        },
      },
      {
        node: {
          id: "QXJ0d29yazo0ZDhiOTM5NTRlYjY4YTFiMmMwMDE5NTA=",
          image: {
            aspect_ratio: 0.7,
            url: "https://d32dm0rphc51dk.cloudfront.net/HKx0sMXcfEzuhrhcHEdUog/large.jpg",
          },
          title: "Constance Towers",
          date: "2009",
          sale_message: null,
          is_biddable: false,
          is_acquireable: false,
          is_offerable: false,
          slug: "john-currin-constance-towers",
          sale: null,
          sale_artwork: null,
          artists: [
            {
              name: "John Currin",
            },
          ],
          partner: {
            name: "Gagosian",
          },
          href: "/artwork/john-currin-constance-towers",
        },
      },
    ],
  },
}
