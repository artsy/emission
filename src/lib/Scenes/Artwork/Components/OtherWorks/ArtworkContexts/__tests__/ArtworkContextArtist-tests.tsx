import { ArtworkFixture } from "lib/__fixtures__/ArtworkFixture"
import { MockRelayRenderer } from "lib/tests/MockRelayRenderer"
import { renderUntil } from "lib/tests/renderUntil"
import React from "react"
import { graphql } from "react-relay"
import { Header } from "../../Header"
import { ArtworkContextArtistFragmentContainer as ArtworkContextArtist } from "../ArtworkContextArtist"

jest.unmock("react-relay")

describe("AllArtists", () => {
  it("renders properly", async () => {
    const tree = await renderUntil(
      wrapper => {
        return wrapper.find(Header).length > 0
      },
      <MockRelayRenderer
        Component={ArtworkContextArtist}
        query={graphql`
          query ArtworkContextArtistTestsQuery($excludeArtworkIds: [String]) {
            artwork(id: "abbas-kiarostami-untitled-7") {
              ...ArtworkContextArtist_artwork
            }
          }
        `}
        mockResolvers={{
          Artwork: () => ArtworkFixture,
        }}
      />
    )

    expect(tree.html()).toMatchSnapshot()
  })
})
