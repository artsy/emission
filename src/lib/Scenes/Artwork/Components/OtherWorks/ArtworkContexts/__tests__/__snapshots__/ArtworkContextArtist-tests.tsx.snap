// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AllArtists renders properly 1`] = `"<view flexdirection=\\"column\\" alignitems=\\"center\\"><text font-family=\\"AGaramondPro-Regular\\" font-size=\\"22px,32px\\" lineheight=\\"28px,38px\\" class=\\"sc-bwzfXH uTxCW\\" mb=\\"2\\" textalign=\\"center\\">Other works by Abbas Kiarostami</text></view>"`;
